api/proto/banyandb/common/v1/common.proto

```diff
@@ -33,9 +34,9 @@ enum Catalog {
 // Metadata is for multi-tenant, multi-model use
 message Metadata {
     // group contains a set of options, like retention policy, max
-    string group = 1;
+    string group = 1 [(validate.rules).string.min_len = 1];
```

Comment:
https://github.com/apache/skywalking-banyandb-java-client/blob/14e2e4f7c0ec5d7a33f567121375194527e5da0d/src/main/java/org/apache/skywalking/banyandb/v1/client/metadata/Group.java#L78

For `Group`, we currently do not duplicate this field. If this rule is imposed, `banyandb-java-client` should also be modified to adapt this change.

Reply:
thank you for your reminding, the group field seems not be mandatory since `Group` only has the `name`. Let's remove this checker here.